### PR TITLE
Add contribs and License

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+2. Create a branch on your fork of the repo with your changes, including a unit test covering the new or modified code.
+3. Submit a PR. The PR description should include a link to the bug on Launchpad.
+4. Update the Launchpad bug to include a link to the PR and the `review-needed` tag.
+5. Once reviewed and merged, the change will become available on the edge channel and assigned to an appropriate milestone
+   for further release according to priority.
+
+## Documentation
+
+Documentation for this charm is currently maintained as part of the Charmed Kubernetes docs.
+See [this page](https://github.com/charmed-kubernetes/kubernetes-docs/blob/master/pages/k8s/charm-docker-registry.md)
```
Comment (addyess) [inline CONTRIBUTING.md:37]: Page has moved right?

```suggestion
See [this page](https://discourse.charmhub.io/t/docker-registry-docs-index/6180)
```

**Review by addyess — CHANGES_REQUESTED** [review]
small nit

**Comment by addyess** [comment]
integration tests are being bitten by: https://github.com/juju/python-libjuju/issues/719

**Comment by addyess** [comment]
@evilnick could you rebase?

**Review by kwmonroe — APPROVED** [review]

**Pull request merged**